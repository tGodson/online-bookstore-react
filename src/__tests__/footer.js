import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/footer/footer';

it('renders footer correctly', () => {
  const tree = renderer.create(<Footer />);
  expect(tree).toMatchSnapshot();
});
