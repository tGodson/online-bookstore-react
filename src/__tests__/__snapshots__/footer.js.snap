// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer correctly 1`] = `
<div
  className="footer"
>
  <p>
    CopyRight 2021Â© Tendongze Godson
  </p>
</div>
`;
